example_site/src/components/App.css

```diff
@@ -35,38 +36,30 @@ main {
   grid-template-columns: repeat(2, 1fr);
 }
 
-.webhook-testing-container {
-  display: grid;
-  grid-template-columns: repeat(2, 1fr);
-}
-
 .test-form-grid {
   display: grid;
   grid-template-columns: repeat(2, 1fr);
 }
 
-.panel {
-  padding: 24px;
-  margin-bottom: 8px;
-  border-radius: 4px;
-  background-color: white;
-  margin: 12px;
-  overflow: auto;
-}
-
 .form-label {
   display: block;
   font-weight: bold;
   margin: 12px 0 4px;
 }
 
+.center {
+  margin: 0 auto;
+  padding-left: 40px;
+  padding-right: 40px;
+}
+
 .submit-button {
```

Comment:
Since we want all buttons to be uppercase, instead of setting text in JS as uppercase, i.e. `SEE HOW IT WORKS`, you can add `text-transform: uppercase;` here. This way the rule is enforced site-wide without you or future developers needing to remember to use uppercase.